samples/tutorials/hardwareInterfaces/adc/NoiseSensor/noiseDataComponent/noiseData.c

```diff
@@ -13,14 +13,11 @@
  * @note Please change this timeout value as needed.
  */
 //--------------------------------------------------------------------------------------------------
-#define ADC_SAMPLE_INTERVAL_IN_SECONDS (1)
+#define ADC_SAMPLE_INTERVAL_IN_MILLI_SECONDS (1)
```

Comment:
I think you should have changed the value to `1000` right?  Also, I milliseconds is one word, so I would prefer `ADC_SAMPLE_INTERVAL_IN_MILLISECONDS` or `ADC_SAMPLE_INTERVAL_IN_MS`.